### ISSUE TITLE
Crash on move

### ISSUE THREAD

**Issue by EmmanuelMess** [TeamAmaze/AmazeFileManager#1173]
Moving ~>200 files from different folders app crashed with:

```gradle
E/AndroidRuntime: FATAL EXCEPTION: AsyncTask #3
    Process: com.amaze.filemanager, PID: 17313
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:309)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:354)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:223)
        at java.util.concurrent.FutureTask.run(FutureTask.java:242)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
        at java.lang.Thread.run(Thread.java:818)
     Caused by: java.lang.IllegalStateException: This is not the first datapoint!
        at com.amaze.filemanager.asynchronous.services.AbstractProgressiveService.addFirstDatapoint(AbstractProgressiveService.java:169)
        at com.amaze.filemanager.asynchronous.services.CopyService$DoInBackground.doInBackground(CopyService.java:216)
        at com.amaze.filemanager.asynchronous.services.CopyService$DoInBackground.doInBackground(CopyService.java:186)
        at android.os.AsyncTask$2.call(AsyncTask.java:295)
        at java.util.concurrent.FutureTask.run(FutureTask.java:237)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588) 
        at java.lang.Thread.run(Thread.java:818) 
```

Notification afterwards is stuck.

**Comment by EmmanuelMess** [comment]
@VishalNehra Any comments?

**Comment by VishalNehra** [comment]
Trying to pinpoint the cause. The same can easily be reproduced while trying to extract a gz or 7z file as implemented in #1170

**Comment by N00byKing** [comment]
Dont know if this helps, but after some testing this did not happen for me on the Android Emulator (The one included in Android Studio), but on my Phone it does (Only sometimes though, for some reason)
